doc/sections/mira-03-microcity.tex

```diff
@@ -189,6 +201,15 @@ \section{Planner}
 	\label{fig:planner-arch}
 \end{figure}
 
+In addition, park activities are as dynamic as park visitors are so it is easy to see that the needs of both parties may change in the process. Therefore, it is appropriate to provide that the plans generated for visitors can change dynamically to adapt to changes in the environment.
```

Comment:
In order to do that we should make a connection with the recommendation system (?) what do you think?